src/include/utils/boost_utils.h

```diff
@@ -0,0 +1,23 @@
+//
+// Created by steven on 2/4/16.
+//
+
```

Comment:
Updated at the next commit.
Both this and call.cc can compile and run without any #include <boost/**>. Is this a bad sign?